backend/api/Controllers/Models/MissionDefinitionResponse.cs

```diff
@@ -6,37 +6,26 @@ namespace Api.Controllers.Models
 {
     public class MissionDefinitionResponse
     {
-        [JsonPropertyName("id")]
         public string Id { get; set; } = string.Empty;
 
-        [JsonPropertyName("name")]
         public string Name { get; set; } = string.Empty;
 
-        [JsonPropertyName("installationCode")]
         public string InstallationCode { get; set; } = string.Empty;
 
-        [JsonPropertyName("comment")]
         public string? Comment { get; set; }
 
-        [JsonPropertyName("inspectionFrequency")]
         public TimeSpan? InspectionFrequency { get; set; }
 
-        [JsonPropertyName("autoScheduleFrequency")]
         public AutoScheduleFrequency? AutoScheduleFrequency { get; set; }
 
-        [JsonPropertyName("lastSuccessfulRun")]
         public virtual MissionRun? LastSuccessfulRun { get; set; }
 
-        [JsonPropertyName("inspectionArea")]
         public InspectionAreaResponse? InspectionArea { get; set; }
 
-        [JsonPropertyName("isDeprecated")]
```

Comment:
I'm a bit confused by this change. Is the reasoning that the specified names were equivalent to what was anyways generated based on the JSON encoder config?

Reply:
This is something we use on messages from isar, sara and Echo. MissionDefinitionResponse is the only other place the JsonPropertyName is used. Removing it cause it is unnecessary, and to increase inconsistency in how we implement models

Reply:
We do use it in the frontend buts it not necessary to specify as Json if the json matches the existing name. It will be the same as what we do with MissionRunResponse